---
title: Genomic analysis of COVID-19 spread in Switzerland. Situation report 2020-06-07.
authors:
  - Emma Hodcroft
  - Richard Neher
  - Sarah Nadeau
  - Tanja Stadler
authorLinks:
  - https://neherlab.org/emma-hodcroft.html/
  - https://neherlab.org/richard-neher.html/
  - https://bsse.ethz.ch/cevo/
  - https://bsse.ethz.ch/cevo/

affiliations: "Biozentrum, Basel, Switzerland; D-BSSE ETHZ Basel"
license: "CC-BY"
licenseLink: "https://creativecommons.org/licenses/by/4.0/"
dataset: "https://nextstrain.org/groups/swiss/ncov/switzerland?d=map&f_country=Switzerland&legend=closed"
date: 2020-June-06
abstract: ""
---

<!-- This is left-side text 1-->
# [Table of Contents]("https://nextstrain.org/groups/swiss/ncov/switzerland/2020-06-07?d=map&f_country=Switzerland&legend=closed")

  * [Diversity of SARS-CoV-2](https://nextstrain.org/groups/swiss/narratives/sit-rep-2020-06-07?n=2).
  * [SARS-CoV-2 in Europe](https://nextstrain.org/groups/swiss/narratives/sit-rep-2020-06-07?n=3).
  * [SARS-CoV-2 in Switzerland](https://nextstrain.org/groups/swiss/narratives/sit-rep-2020-06-07?n=4).
  * [Putative clusters in Switzerland](https://nextstrain.org/groups/swiss/narratives/sit-rep-2020-06-07?n=5).

We've prepared some resources that will make interpreting the data we present in this narrative easier.

  * [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).
  * [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
  * [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

 * SARS-CoV-2 emerged in China in late 2019. Epidemiological, diagnostic, and genetic evidence clearly support a single spill-over in late 2019
 * SARS-CoV-2 had spread globally by mid-February and established outbreaks across the world.
 * Sequences sampled in Switzerland are diverse and representative of European diversity.
 * Sparse sampling makes reconstruction of transmission clusters difficult, but with more diversity and denser sampling phylogenetics can complement contact tracing.
```


<!-- ############ SLIDE BREAK ############# -->
# [Diversity of SARS-CoV-2](https://nextstrain.org/groups/swiss/ncov/switzerland/2020-06-07?c=clade_membership)

 * From the initial SARS-CoV-2 clades 19A and 19B, several notable new clades have emerged in 2020
 * Most clade are present on every continent, but the relative abundance varies from region to region
 * Clade 20A and its descendants 20B and 20C have gradually taken over as the pandemic moved from Asia to Europe and the Western and Southern hemisphere. The spread of these clades is more likely due to chance than to intrinsic differences between the viral variants.

# [Diversity of SARS-CoV-2 in Europe](https://nextstrain.org/groups/swiss/ncov/switzerland/2020-06-07?c=clade_membership&f_region=Europe)

 * Clade 20A and 20B dominate Europe since March, while other clades were more common early during the outbreak
 * But also variants 19A and 19B not associated with the large North Italian outbreak continue to circulate in the UK, Spain, and central European countries.

# [SARS-CoV-2 in Switzerland](https://nextstrain.org/groups/swiss/switzerland/2020-06-07?c=clade_membership&f_country=Switzerland)

 * The tree on the right shows individual samples at their collection date.
 * The earliest available genomes were sampled on February 24, the lastest on April 7.
 * Consistent with the broader European picture, sequences from Switzerland sampled in March and April fall into clades 20A, 20B, and 20C. Only a few samples fall into clades 19A and 19B.

# [SARS-CoV-2 in Switzerland](https://nextstrain.org/groups/swiss/switzerland/2020-06-07?c=gt-nuc_3116&f_country=Switzerland&label=clade:20A&m=div)

 * This rendering of the tree shows genetic rather than temporal distance between samples and zoomed into the European outbreak clade 20A.
 * In the early phase of the European outbreak, diversity was limited and many sequences are identical (i.e. they sit on the same vertical bar).
 * This low diversity limits the resolution at which transmission clusters can be resolved.
 * A small subclade with mutation A3116G (yellow color) is restricted to the greater Zürich area and might correspond to a local transmission cluster.
 * With increasing diversity over time, the resolution of genomic epidemiology will increase.


<!-- This is left-side text 12-->
# [Scientific credit]("https://nextstrain.org/groups/swiss/ncov/switzerland/2020-06-07?d=map&f_country=Switzerland&legend=closed")

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.
<br><br>
The data from Switzerland was contributed by the following groups:

 * [Computational Evolution group](https://bsse.ethz.ch/cevo) at D-BSSE of the ETHZ in Basel. Samples were provided by [Viollier AG](www.viollier.ch), sequenced on Illumina technology by the Genomic Facility Basel (lead [C. Beisel](https://bsse.ethz.ch/genomicsbasel)), and processed with the [V-Pipe viral assembly pipeline](https://cbg-ethz.github.io/V-pipe/sars-cov-2/) ([Ivan Topolsky, Pedro Ferreira, Philipp Jablonski, Susana Posada-​Céspedes, Niko Beerenwinkel](https://bsse.ethz.ch/cbg)).
 * University Hospital Basel, [Clinical Virology](https://www.unispital-basel.ch/ueber-uns/bereiche/medizinische-querschnittsfunktionen/kliniken-institute-abteilungen/labormedizin/ueber-uns/klinische-virologie/kontakt-und-oeffnungszeiten/) and [Clinical Microbiology](https://www.unispital-basel.ch/ueber-uns/bereiche/medizinische-querschnittsfunktionen/kliniken-institute-abteilungen/labormedizin/ueber-uns/klinische-bakteriologie-mykologie/kontakt-und-oeffnungszeiten/).
 * [Laboratory of Virology (University Hospitals of Geneva (HUG))](https://www.hug-ge.ch/laboratoire-virologie). Samples were collected from different cantons in Switzerland, sequenced on Illumina HiSeq4000 instrument by the iGE3 Genomics Platform from the University of Geneva (lead M.Docquier) and processes by the Laboratory of Virology from HUG (Florian Laubscher, Samuel Cordey and Laurent Kaiser).
 * [Institute of Medical Virology at University of Zurich](https://www.virology.uzh.ch/de/research/gtrkolad.html). Samples were provided by University Hospital Zurich, Triemli Hospital Zurich and Klinik Hirslanden Zurich, sequenced on Illumina technology and processed with VirMet viral metagenomic pipeline (Stefan Schmutz, Maryam Zaheri, Verena Kufner, Gabriela Ziltener, Jürg Böni, Michael Huber, Alexandra Trkola).

<br><br>
The specific sequences provided by each lab are [listed here](https://github.com/nextstrain/ncov/blob/e8020ca13e420bb62e31acf9a07268bc08b11c63/narratives/sit-rep_credits.md).
<br><br>
We also gratefully acknowledge GISAID for providing the platform through which these data can be uploaded and shared.
